File: android/RecyclerView-master/MyFirebaseCloudMessaging/app/src/main/java/com/dicoding/myfirebasecloudmessaging/MainActivity.java
package com.dicoding.myfirebasecloudmessaging;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.Button;
import android.widget.Toast;

import com.google.firebase.messaging.FirebaseMessaging;

public class MainActivity extends AppCompatActivity {
    private static final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button logTokenButton = findViewById(R.id.btn_token);
        logTokenButton.setOnClickListener(v ->
                FirebaseMessaging.getInstance().getToken().addOnSuccessListener(deviceToken -> {
                    String msg = getString(R.string.msg_token_fmt, deviceToken);
                    Toast.makeText(MainActivity.this, msg, Toast.LENGTH_SHORT).show();
                    Log.d(TAG, "Refreshed token: " + deviceToken);
                })
        );
    }
}

File: android/RecyclerView-master/GithubUserApp/app/src/main/java/com/dicoding/githubuserapp/MainActivity.kt
package com.dicoding.githubuserapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.ListView

class MainActivity : AppCompatActivity() {

    private var usernameGithub = resources.getStringArray(R.array.username)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val listView:ListView = findViewById(R.id.lv_list)
        val adapter = ArrayAdapter<String>(this,
            android.R.layout.simple_list_item_1, android.R.id.text1, usernameGithub)
        listView.adapter = adapter
    }

}